demosplan/DemosPlanCoreBundle/Logic/Import/Statement/StatementSpreadsheetImporterWithZipSupport.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace demosplan\DemosPlanCoreBundle\Logic\Import\Statement;
+
+use DemosEurope\DemosplanAddon\Contracts\Entities\ProcedureInterface;
+use demosplan\DemosPlanCoreBundle\Doctrine\Generator\NCNameGenerator;
+use demosplan\DemosPlanCoreBundle\Entity\File;
+use demosplan\DemosPlanCoreBundle\Entity\User\User;
+use demosplan\DemosPlanCoreBundle\Exception\CopyException;
+use demosplan\DemosPlanCoreBundle\Exception\DemosException;
+use demosplan\DemosPlanCoreBundle\Exception\InvalidDataException;
+use demosplan\DemosPlanCoreBundle\Exception\MissingPostParameterException;
+use demosplan\DemosPlanCoreBundle\Exception\StatementElementNotFoundException;
+use demosplan\DemosPlanCoreBundle\Exception\UnexpectedWorksheetNameException;
+use demosplan\DemosPlanCoreBundle\Exception\UserNotFoundException;
+use demosplan\DemosPlanCoreBundle\Logic\Document\ElementsService;
+use demosplan\DemosPlanCoreBundle\Logic\FileService;
+use demosplan\DemosPlanCoreBundle\Logic\Procedure\CurrentProcedureService;
+use demosplan\DemosPlanCoreBundle\Logic\Statement\StatementCopier;
+use demosplan\DemosPlanCoreBundle\Logic\Statement\StatementService;
+use demosplan\DemosPlanCoreBundle\Logic\StatementAttachmentService;
+use demosplan\DemosPlanCoreBundle\Logic\User\CurrentUserService;
+use demosplan\DemosPlanCoreBundle\Logic\User\OrgaService;
+use demosplan\DemosPlanCoreBundle\Logic\ZipImportService;
+use Doctrine\ORM\EntityManagerInterface;
+use InvalidArgumentException;
+use Symfony\Component\Finder\SplFileInfo;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Webmozart\Assert\Assert;
+
+class StatementSpreadsheetImporterWithZipSupport extends StatementSpreadsheetImporter
+{
+    private ?array $fileMap;
+    public function __construct(
+        CurrentProcedureService $currentProcedureService,
+        CurrentUserService $currentUser,
+        ElementsService $elementsService,
+        OrgaService $orgaService,
+        StatementCopier $statementCopier,
+        StatementService $statementService,
+        TranslatorInterface $translator,
+        ValidatorInterface $validator,
+        private readonly ZipImportService $zipImportService,
+        private readonly FileService $fileService,
+        private readonly NCNameGenerator $nameGenerator,
```

Comment:
never used

Reply:
thx, removed in 36ce9a5f5e2953295255fade8243a10dbfc8b510